### PR TITLE
[HNC-501] - Create stable tag for latest tagged release in gh-composite actions.

### PR DESCRIPTION
**Pull request by mayur-hitachivantara**
useful links:-

When a stable tag already exists then it updates the commit associated with the stable tag to the latest commit [link](https://github.com/lumada-common-services/lds-composite-action/actions/runs/6941822831/job/18883448250#step:6:152).
When a stable tag doesn't exist then creates new one with latest commit associated to stable tag [link](https://github.com/lumada-common-services/lds-composite-action/actions/runs/6942319979/job/18885028030#step:6:153)

### CODE REVIEW

File: .github/workflows/update-release-notes.yml [hunk 1]
```diff
@@ -12,8 +12,26 @@ env:
...
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: master
+          token: ${{ secrets.HV_ADMIN_GITHUB_TOKEN}} 
+      
+      - run: sudo chmod u+rwx tag.sh
```
Comment (cardosov) [inline .github/workflows/update-release-notes.yml:26]: Do you need `sudo`? `chmod +x tag.sh` is not enough?

**Review by cardosov — CHANGES_REQUESTED** [review]

File: .github/tag.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        git tag -f $stable_tag_name $latest_commit
+    fi
+else
+    # Create a new static tag
+    echo "Creating a new stable tag '$stable_tag_name'..."
+    git tag $stable_tag_name $latest_commit
+fi
+
+# Push the updated or new tag
+git push -f --tags
```
[the remark this answers is not in the document]
Comment (mayur-hitachivantara) [reply .github/tag.sh:40]: Yes, Update push for static tag only

File: .github/workflows/update-release-notes.yml (hunk 1 above)
Comment (mayur-hitachivantara) [reply .github/workflows/update-release-notes.yml:26]: No, `chmod +x tag.sh` works fine

File: .github/workflows/update-release-notes.yml [hunk 3]
```diff
@@ -12,8 +12,26 @@ env:
...
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: master
+          token: ${{ secrets.HV_ADMIN_GITHUB_TOKEN}} 
+      
+      - run: chmod u+rwx entrypoint.sh
```
Comment (cardosov) [inline .github/workflows/update-release-notes.yml:26]: Don't mean `tags.sh` and just `chmod +x`?

**Review by cardosov — CHANGES_REQUESTED** [review]

**Review by cardosov — APPROVED** [review]

**Review by AdityaHitachi — APPROVED** [review]

**Pull request merged**